ci/actions-templates/README.md

```diff
@@ -35,43 +35,44 @@ cross-build we don't run the tests. All the builds which aren't mac or windows
 system.
 
-| Target                        | Cross      | Tier  | On PR? | On master? |
-| ----------------------------- | ---------- | ----- | ------ | ---------- |
-| x86_64-unknown-linux-gnu      | No         | One   | Yes    | Yes        |
-| armv7-unknown-linux-gnueabihf | Yes        | Two   | Yes    | Yes        |
-| aarch64-linux-android         | Yes        | Two   | Yes    | Yes        |
-| aarch64-unknown-linux-gnu     | Yes        | Two   | No     | Yes        |
-| aarch64-unknown-linux-musl    | Yes        | Two   | No     | Yes        |
-| powerpc64-unknown-linux-gnu   | Yes        | Two   | No     | Yes        |
-| x86_64-unknown-linux-musl     | Yes        | Two   | No     | Yes        |
-| i686-unknown-linux-gnu        | Yes        | One   | No     | No         |
-| arm-unknown-linux-gnueabi     | Yes        | Two   | No     | No         |
-| arm-unknown-linux-gnueabihf   | Yes        | Two   | No     | No         |
-| x86_64-unknown-freebsd        | Yes        | Two   | No     | No         |
-| x86_64-unknown-netbsd         | Yes        | Two   | No     | No         |
-| x86_64-unknown-illumos        | Yes        | Two   | No     | No         |
-| powerpc-unknown-linux-gnu     | Yes        | Two   | No     | No         |
-| powerpc64le-unknown-linux-gnu | Yes        | Two   | No     | No         |
-| mips-unknown-linux-gnu        | Yes        | Two   | No     | No         |
-| mips64-unknown-linux-gnu      | Yes        | Two   | No     | No         |
-| mipsel-unknown-linux-gnu      | Yes        | Two   | No     | No         |
-| mips64el-unknown-linux-gnu    | Yes        | Two   | No     | No         |
-| s390x-unknown-linux-gnu       | Yes        | Two   | No     | No         |
-| arm-linux-androideabi         | Yes        | Two   | No     | No         |
-| armv7-linux-androideabi       | Yes        | Two   | No     | No         |
-| i686-linux-android            | Yes        | Two   | No     | No         |
-| x86_64-linux-android          | Yes        | Two   | No     | No         |
-| riscv64gc-unknown-linux-gnu   | Yes        | ---   | No     | No         |
-| loongarch64-unknown-linux-gnu | Yes        | Two   | No     | No         |
-| ----------------------------- | ---------- | ----- | ------ | ---------- |
-| aarch64-apple-darwin          | Yes        | Two   | Yes    | Yes        |
-| x86_64-apple-darwin           | No         | One   | Yes    | Yes        |
-| ----------------------------- | ---------- | ----- | ------ | ---------- |
-| x86_64-pc-windows-msvc        | No         | One   | Yes    | Yes        |
-| x86_64-pc-windows-gnu         | No         | One   | No     | Yes        |
-| i686-pc-windows-msvc          | No         | One   | No     | No         |
-| i686-pc-windows-gnu           | No         | One   | No     | No         |
-| aarch64-pc-windows-msvc       | Yes        | Two   | No     | Yes        |
+| Target                         | Cross      | Tier  | On PR? | On master? |
```

Comment:
@djc Wait. I think this table, together with the paragraph above, is largely outdated and many parts of the info provided here are no longer true. Should we remove all this altogether and see the CI YMLs as the single source of truth?

Reply:
Makes sense to me.

Reply:
@djc I've replaced the original with a short section explaining our platform support policy.